pom.xml

```diff
@@ -520,6 +520,9 @@
               <!-- docs (website) build target dir -->
               <exclude>docs/_site/**</exclude>
               <exclude>docs/Gemfile.lock</exclude>
+
+	      <!-- compiled R packages (binaries) -->
+	      <exclude>R/**</exclude>
```

Comment:
Does not this exclude all `./r/` maven sub-module? (assuming cases-insensitive FS)?
AFAIK compiled R packages now live under `~/R`